sdb/commands/print.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright 2020 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+import argparse
+
+import drgn
+import sdb
+
+
+class Print(sdb.SingleInputCommand):
+    """
+    Format output for shell script consumption.
+
+    EXAMPLES
+        By default getting the address of a global variable
+        looks like this:
+
+            sdb> addr spa_namespace_avl
+            (avl_tree_t *)spa_namespace_avl+0x0 = 0xffffffffc0954fe0
+
+        One could use the `print` command to format the above
+        to something that is better for consumption like this:
+
+            sdb> addr spa_namespace_avl | print -nr
+            0xffffffffc0954fe0
+    """
+
+    names = ["print"]
+
+    @classmethod
+    def _init_parser(cls, name: str) -> argparse.ArgumentParser:
+        #
+        # Check the following for available options that are built-in to drgn:
+        # drgn.readthedocs.io/en/latest/api_reference.html#drgn.Object.format_
+        #
+        parser = super()._init_parser(name)
+        parser.add_argument("-c",
+                            "--char",
+                            action="store_true",
+                            default=False,
```

Comment:
I think that `default` may not be needed here, since according to the docs:
> 'store_true' and 'store_false' - These are special cases of 'store_const' used for storing the values True and False respectively. In addition, they create default values of False and True respectively.

Reply:
You are right! Thanks for pointing that out